components/admin/requests/physician-assessment/Card.tsx

```diff
@@ -0,0 +1,139 @@
+import { formatDate } from '@lib/utils/format';
+import { titlecase } from '@tools/string';
+
+type Props = {
+  readonly applicationId: number;
+  readonly isUpdated?: boolean;
+  /** Whether card is a subsection */
+  readonly isSubsection?: boolean;
+};
+
+const Card: FC<Props> = props => {
+  const { applicationId, isUpdated, isSubsection } = props;
+
+  const [physicianAssessment, setPhysicianAssessment] = useState<PhysicianAssessment | null>(null);
+
+  useQuery<GetPhysicianAssessmentResponse, GetPhysicianAssessmentRequest>(
+    GET_PHYSICIAN_ASSESSMENT,
+    {
+      variables: { id: applicationId },
+      onCompleted: data => {
+        if (data) {
+          setPhysicianAssessment(data.application);
+        }
+      },
+      notifyOnNetworkStatusChange: true,
+    }
+  );
+
+  if (!physicianAssessment) {
+    return null;
+  }
+
+  const {
+    disability,
+    disabilityCertificationDate,
+    patientCondition,
+    otherPatientCondition,
+    temporaryPermitExpiry,
+    permitType,
+  } = physicianAssessment;
+  return (
+    <PermitHolderInfoCard
+      colSpan={7}
+      header={`Physician’s Assessment`}
+      updated={isUpdated}
+      divider
+      isSubsection={isSubsection}
+      editModal={false}
+    >
+      <VStack align="left" spacing="12px">
+        <Grid
+          gridRowGap="12px"
+          gridColumnGap="20px"
+          templateColumns="200px 1fr"
+          gridAutoRows="minmax(32px, auto)"
+        >
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Medical name of disabling condition(s)
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              {disability}
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Certification date
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              {formatDate(disabilityCertificationDate)}
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Condition
+            </Text>
+          </GridItem>
+          <Wrap>
+            <Badge bgColor="background.informative">{titlecase(patientCondition as string)}</Badge>
+          </Wrap>
+          {patientCondition === 'OTHER' && (
+            <>
+              {/* Condition description */}
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  Condition description
+                </Text>
+              </GridItem>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  {otherPatientCondition}
+                </Text>
+              </GridItem>
+            </>
+          )}
```

Comment:
I wanted to confirm if the condition description row is only rendered when the patient condition is `Other`. I'll ask designers when I ping them for a design review

Reply:
Yes, that's my understanding, cc @carelynntsai @vedantdsn to confirm

Reply:
yep that's correct!